console: Document oidc issuer url and tls config (#771)
redpanda-data/documentation: docs/console/single-sign-on/identity-providers/generic-oidc.mdx

    # and the `issuer` returned in the response has to match this issuer url.
    issuerUrl: ""

>     # IssuerTLS is the TLS config that is used by the HTTP client to send requests
> 	  # to the IssuerURL. If you don't set any certificate paths, this will default to 
>     # the system cert pool.

Reviewer: ```suggestion
    # IssuerTLS is the TLS configuration used by the HTTP client to send requests
	  # to the IssuerURL. If you don't set any certificate paths, the IssuerTLS defaults to 
    # the system cert pool.
```

    issuerTls:
      caFilepath:
      certFilepath:
      keyFilepath:

>     # DisplayName is the name that shall be shown on the login page for this identity provider

Reviewer: ```suggestion
    # DisplayName is the name that will be shown on the login page for this identity provider
```
